dotCMS/src/main/java/com/dotcms/cms/login/LoginServiceAPIFactory.java

```diff
@@ -333,6 +333,29 @@ private void doAuthentication(String userId, boolean rememberMe, HttpServletRequ
 
             ses.removeAttribute("_failedLoginName");
 
+            
+            
+
+            
+            if(Config.getBooleanProperty("PREVENT_SESSION_FIXATION_ON_LOGIN", true)) {
```

Comment:
very nice, we did something like that on SAML.
I would recommend to put it into a protected method such as 
preventSessionFixationOnLogin :)